server/common/src/main/java/org/eclipse/lsp/cobol/common/model/tree/SectionNameNode.java

```diff
@@ -58,7 +58,7 @@ public List<MarkupContent> getFormattedDisplayString() {
         getNearestParentByType(NodeType.PROCEDURE_SECTION)
             .map(ProcedureSectionNode.class::cast)
             .map(ProcedureSectionNode::getFullVariableDescription)
-            .map(desc -> new MarkupContent(MarkupKind.MARKDOWN, desc))
+            .map(desc -> new MarkupContent(MarkupKind.PLAINTEXT, desc))
             .orElse(new MarkupContent(MarkupKind.MARKDOWN, "")));
```

Comment:
Is the orElse empty markdown somehow significant or should the code also do
```
        .orElse(Stream.empty())
        .map(line -> new MarkupContent(MarkupKind.PLAINTEXT, line))
        .collect(Collectors.toList());
```
just like for the paragraphnamenode?

Reply:
Updated.